S06-signature/types.t

```diff
@@ -35,4 +35,14 @@ dies-ok  { g('m') },    'type check forbids bad implicit return';
         'no segfault when calling a routine having a native parameter with a type object argument';
 }
 
+# RT #126124
+{
+    throws-like { sub f(Mu:D $a) {}; f(Int) }, Exception,
+        message => 'Parameter \'$a\' requires an instance of type Mu, but a type object was passed.  Did you forget a .new?',
+        'type shown in the exception message is the right one';
+    throws-like { sub f(Mu:U $a) {}; f(123) }, Exception,
+        message => 'Parameter \'$a\' requires an instance of type Mu, but a type object was passed.  Did you forget a .new?',
```

Comment:
Seems this line expects the wrong message. If the test is passing, something wrong, because the message should contain `but an object instance was passed`, yet the test tests for `but a type object was passed`